tests/drivers/regulator/fixed/prj.conf

```diff
@@ -1,5 +1,7 @@
 CONFIG_GPIO=y
 CONFIG_REGULATOR=y
+# Explicitly enable REGULATOR_FIXED as workaround for
+# "No SOURCES given to target:  drivers__regulator" CMake error.
```

Comment:
please check: https://github.com/zephyrproject-rtos/zephyr/pull/34289
for a more general fix that covers the empty Zephyr lib in general for drivers.

Reply:
rebased and removed comment

Reply:
@jfischer-no #34289 has been merged, please update the prj.conf to not contain workarounds, so we can see if there are any other outstanding issues in the build system.